packages/block-editor-tools/package.json

```diff
@@ -45,6 +45,7 @@
   "homepage": "https://github.com/alleyinteractive/alley-scripts#readme",
   "dependencies": {
     "@alleyinteractive/tsconfig": "*",
+    "@storybook/react": "^7.5.1",
```

Comment:
Needed to address ts errors

Reply:
This is needed in the published package dependencies, not the dev dependencies? That seems peculiar.

Reply:
Yeah, not sure I understand it 100% either, but `npm run build` wouldn't work without that

Reply:
Actually moved to dev dependencies and is fine, thanks!